letsencrypt/scripts/main.py

```diff
@@ -215,5 +220,23 @@ def read_file(filename):
         raise argparse.ArgumentTypeError(exc.strerror)
 
 
+def split_tsig_keys(packed):
+    """Returns the unpacked TSIG key name, secret, and a list of domains from
+    the argument format.
+
+    :param str packed: Packed
+
+    :returns: A tuple of key name, secret, and a list of domains
+    :rtype: tuple
+
+    :raises argparse.ArgumentTypeError: Packed TSIG key is in incorrect format."""
```

Comment:
pls use the following style (empty line between end of comment and docstring ending markers):
```

... in incorrect format.

"""
```